client/blocks/hello-vote-notice/index.jsx

```diff
@@ -70,6 +72,10 @@ export default connect(
 				isSiteSection( state ) &&
 				'settings' !== getSectionName( state ) &&
 				canCurrentUser( state, selectedSiteId, 'manage_options' ) &&
+				(
+					! userStartDate ||
+					moment( userStartDate ).isBefore( moment().subtract( 1, 'weeks' ) )
+				) &&
```

Comment:
>You can use `isNewUser` from [Guided Tours contexts](https://github.com/Automattic/wp-calypso/blob/c256a69bd15a139dc06cb446f28b53aab9a60125/client/state/ui/guided-tours/contexts.js#L20) :)

This seems like something that should be in `state/current-user/selectors.js`... or, y'know, we could go the whole global selectors route 😄  #5954

I kinda like the `moment` implementation @nylen has here. Not sure if it's any less performant, but it reads nicely at least.